### DISCUSSION TITLE
Unable to install a specific version of our CLI using scoop

### DISCUSSION THREAD

**Discussion by maidul98** [goreleaser/goreleaser#3869] [Q&A] [answered]
We have scoop set up so that our CLI can be installed on windows. All is well, but I noticed that we cannot go back to a old version of the CLI. Doing something like `scoop install [app]@[version]` does not work. It says 

```
[XXXX] does not have autoupdate capability
couldn't find manifest for [XXXX]
```

Here is our manifest created by go releaser: https://github.com/Infisical/scoop-infisical/blob/main/infisical.json

I think we need to set some kind of autoupdate capability? I didn't see anything about autoupdate capability in the docs here so any help will be appreciated https://goreleaser.com/customization/scoop/ 

Thanks

**Comment by caarlos0** [comment, marked as answer]
i don't think that autoupdate does that.

see previous discussion at: https://github.com/goreleaser/goreleaser/issues/3713
